src/App.tsx

```diff
@@ -5,6 +5,7 @@ import { Authenthication } from './pages/Authenthication';
 import { Invoices } from './pages/Invoices';
 import { AuthenticationPrivateRoutes } from './components/form-authentication-components/AuthenticationPrivateRoutes';
 import { ThemeProvider } from './context/ThemeContext';
+import InvoiceInfo from './pages/InvoiceInfo';
```

Comment:
https://dev.to/shacodes/default-vs-named-exports-19hj lepiej uzywac named exports
+ poczytaj o lazy loadingu i sprobuj zaimplementowac (nie musi byc w tej pr, ale im szybciej tym mniej roboty)